src/sources/MVT.js

```diff
@@ -42,7 +42,6 @@ export default class MVT extends Base {
             metadata = new MVTMetadata(metadata);
         }
         this._metadata = metadata;
-        this._metadata.setCodecs();
```

Comment:
What problem was causing this?

Reply:
The MVT category codec object has a reference to the metadata, and the metadata has a reference to the codec.

On the `_clone` method, JSON.stringify throw due to a circular structure.